### PR TITLE
chore: Adding NodeJS SDK to FEATURES.md & style fixes.

### PR DESCRIPTION
**Pull request by domderen**

### CODE REVIEW

File: docs/FEATURES.md [hunk 1]
```diff
@@ -48,3 +48,4 @@ alpha/beta/stable.
 | Terminating pipelines | v0.0.59 | v0.0.70 | |
 | Terminating steps | v0.0.59 | v0.0.70 | |
 | User interface | | v0.0.59 | |
+| NodeJS SDK | | v0.0.59 | |
```
Comment (alexec) [inline docs/FEATURES.md:51]: alphabetic order!

Comment (alexec) [reply docs/FEATURES.md:51]: also, should be v0.0.78

Comment (alexec) [reply docs/FEATURES.md:51]: and should be v0.0.78 in the first (alpha) column

Comment (domderen) [reply docs/FEATURES.md]: Updated

**Pull request merged**